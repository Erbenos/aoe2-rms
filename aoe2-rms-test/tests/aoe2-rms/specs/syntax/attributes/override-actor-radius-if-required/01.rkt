#lang aoe2-rms
; Place a barracks with the default actor_area_radius of 1, and then place a house adjacent, and prevent it from overlapping the barracks.

<OBJECTS-GENERATION>
(create-object 'BARRACKS
               (set-place-for-every-player)
               (find-closest)
               (actor-area 2))

(create-object 'HOUSE
               (set-place-for-every-player)
               (actor-area-to-place-in 2)
               (override-actor-radius-if-required))
