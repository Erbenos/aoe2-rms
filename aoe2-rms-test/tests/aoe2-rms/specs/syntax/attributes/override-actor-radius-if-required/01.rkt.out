<OBJECTS_GENERATION>
create_object BARRACKS {
  set_place_for_every_player
  find_closest
  actor_area 2
}
create_object HOUSE {
  set_place_for_every_player
  actor_area_to_place_in 2
  override_actor_radius_if_required
}
